### ISSUE TITLE
Deskband not showing correctly for RTL languages

### ISSUE THREAD

**Issue by srwi** [dsafa/CSDeskBand#54]
I received a [bug report](https://github.com/stnkl/EverythingToolbar/issues/15) on a project of mine where I'm using CSDeskBand. It looks like right to left languages like Hebrew cause the deskband scaling to behave incorrectly. I tested it on my project and Audio Band as you can see in the video. Both of them use CSDeskBand.

![fQdcCMOSd6](https://user-images.githubusercontent.com/17520641/99277701-669b6a00-282e-11eb-969e-9f55480bee3f.gif)

Edit: Only WPF is affected by this.

**Comment by eladpotok** [comment]
try set fix width for the tool, like this

`Options.MinHorizontalSize = new DeskBandSize(27, 40);`

BTW - How did you overcome on the WIN message
![image](https://user-images.githubusercontent.com/32292032/100351158-aa8f2b80-2ff3-11eb-821c-0727649c4490.png)


תודה.

**Comment by srwi** [comment]
Are you also using version 3.1? Does your deskband not have the problem with RTL languages? I actually did set a minimal deskband size like this.

I think you can disable the message by setting [`ShowDeskBand`](https://github.com/dsafa/audio-band/blob/f13b2f226991e46704684e060873e03b26441f0e/src/AudioBand/Deskband.cs#L24) to `false` or just removing the parameter ([like I did](https://github.com/stnkl/EverythingToolbar/blob/6f3ccb5ab4bb1b9cd914ffcbb15c56fb4d0b627f/EverythingToolbar/Deskband.cs#L14)).

**Comment by eladpotok** [comment]
regarding to your question, I am not sure what version I am using since I am using the code and not the nugget package.
I am using WPF host control and set a fix size in the XAML as well. 

About my  question, I did what you suggested. So while registration (by the BAT script) the toolbar is not shown. this is what this boolean parameter does. But I call to ShowDaskBand from another program, this message is shown.

**Comment by srwi** [comment]
I am also using the code file linked in the readme which points to version 3.1 [as indicated](https://github.com/dsafa/CSDeskBand/blob/968bc1aaae12e2587b7bb577c0da86aaa2ff73a3/output/CSDeskBand.cs#L3) in the first few lines of the file. Interesting that you are not experiencing that problem as @dsafa's own project audio-band has it.

Other than that parameter I honestly don't know what is causing that. I don't remember specifically changing anything, sorry.

**Comment by eladpotok** [comment]
Actually My windows taskbar is Left To Right, So I Can't tell if it is fixed or not.
In your project, when you show the taskbar, this message is not shown?

**Comment by srwi** [comment]
No. I don't automatically trigger the taskbar to be shown though. I tell the user to manually enable it via the taskbar context menu. Is that it?

**Comment by eladpotok** [comment]
If I enable the toolbar via the taskbar I don't get this message. I am trying to find an option to disable this message when we enable the toolbar by other program.

**Comment by srwi** [comment]
It might be a safety measure to prevent software from secretly running code from within explorer.exe. That's only a wild guess though.

Edit: https://stackoverflow.com/a/8137927/1477251

**Comment by do2or** [comment]
i confirm that this problem it's still persist in RTL languages
